cli/src/base/StakingCommandBase.ts

```diff
@@ -0,0 +1,91 @@
+import ExitCodes from '../ExitCodes'
+import { AccountId } from "@joystream/types/common"
+import { StakingLedger } from '@polkadot/types/interfaces'
+import AccountsCommandBase from "./AccountsCommandBase"
+import { formatBalance } from '@polkadot/util'
+
+export default abstract class StakingCommandBase extends AccountsCommandBase {
+  
+  private async availableControllers(): Promise<[AccountId,StakingLedger][]> {
+    const controllers = await this.getStakingLedgers()
+    const stakerMap: [AccountId,StakingLedger][] = []
+    controllers.forEach(([key,value]) => {
+      const ledger = value.unwrapOr(undefined)
+      if (!!ledger && this.isKeyAvailable(key)) {
+        stakerMap.push([key,ledger])
+      }
+    })
+    return stakerMap
+  }
+ 
+
+
+  private async getStakingLedgers() {
+    const controllers = await this.getApi().allStakingLedgers()
+    return controllers
+  }
+
+  async getController():Promise<string> {
+    const availableControllers = await this.availableControllers()
+    const selectedValidator = await this.promptForController('No controller input',availableControllers)
```

Comment:
Why is the prompt message set to `No controller input`?

Reply:
Good question!